imgProc_sm.m

```diff
@@ -155,9 +155,17 @@ function imgProc_sm(infile, outfile, probename, n, nEvery, projectname, varargin
 warning off all
 diode_stats = zeros(1,diodenum);
 
-if strcmp(projectname, 'PECAN')  % For example for PECAN dataset 
-    disp('Testing...')  %% Add project specific code if you like
-end
+%if strcmp(projectname, 'PECAN')  % For example for PECAN dataset 
+%    if strcmp(probename, 'PIP')
+%        TimeOffsetSec = -12*3600.0;
```

Comment:
I originally had the `TimeOffsetSec` variable fed in by the `runSizeDist_example.m` script, but opted to streamline things in this branch for the sake of universalness. However, upon testing it there were a number of bugs that still need working out. You can feel free to remove this from the pull request for now.